azure_arc_k8s_jumpstart/cluster_api/capi_azure/arc_capi_azure.sh

```diff
@@ -36,7 +36,45 @@ clusterctl config cluster $CAPI_WORKLOAD_CLUSTER_NAME \
   --kubernetes-version v$KUBERNETES_VERSION \
   --control-plane-machine-count=$CONTROL_PLANE_MACHINE_COUNT \
   --worker-machine-count=$WORKER_MACHINE_COUNT \
+  --infrastructure=$CAPI_PROVIDER \
   > $CAPI_WORKLOAD_CLUSTER_NAME.yaml
+
+wget -O audit.yaml https://raw.githubusercontent.com/Azure/Azure-Security-Center/master/Pricing%20%26%20Settings/Defender%20for%20Kubernetes/audit-policy.yaml
```

Comment:
I will change this to cURL in case someone doesn't have _wget_

Reply:
done